llvm/test/CodeGen/AArch64/GlobalISel/lower-neon-vector-fcmp.mir

```diff
@@ -264,15 +271,16 @@ body:             |
   bb.0:
     liveins: $q0, $q1
 
-    ; Two compares.
```

Comment:
Losing these useful comments

Reply:
That's the downside of using update_mir_test_checks.py.  I see no value in having to manually maintain things that update_mir_test_checks.py has chosen to throw away.  That said, I also don't see them as useful because the CHECK lines already show what is expected. Some of the removed comments were just incorrect.

Reply:
Done.